docs/administration.building-blocks.md

```diff
@@ -27,6 +27,20 @@ In the Admin Area, click on the "Building Block" item in the sidebar under "Plat
 
 > meshStack supports Terraform up to version 1.5.5 and supports OpenTofu versions starting with 1.6.0.
 
+### Building Block Types
+
+The first question you will be prompted is what type of Building Block Definition you want to create. There are
+currently two options:
+
+- **Workspace Building Block**: This is a Building Block that is tied to just a workspace. It is globally available
+  to the entire workspace it was created in. We recommend using this if you have a service that needs to be globally
+  available does not tie to a certain cloud tenant. For example, project tooling or a central identity service.
+- **Tenant Building Block**: This is a Building Block that is tied to a tenant. It is only available to the tenant
+  it was created in. We recommend using this if you have a service that is tied to a certain cloud tenant. For example,
```

Comment:
again the second sentence is repetitive. It would make sense to mention that when the tenant is deleted the building block needs to be deleted as well